force-app/main/default/lwc/marketingCloudCopadoDataTable/marketingCloudCopadoDataTable.js

```diff
@@ -416,40 +412,36 @@ export default class MarketingCloudCopadoDataTable extends LightningElement {
       });
   }
 
- 
   onHandleSort(event) {
     this.sortedBy = event.detail.fieldName;
     this.sortDirection = event.detail.sortDirection;
     this.sortData(this.sortedBy, this.sortDirection);
   }
 
-  sortData(fieldname, direction) { 
-    let parseData='';
-    // 68 sort the data 
-    if(this.data){
+  sortData(fieldname, direction) {
+    let parseData = "";
+    // 68 sort the data
+    if (this.data) {
       parseData = JSON.parse(JSON.stringify(this.data));
-      this.data=null;
-    }else{
+      //this.data = null;
+    } else {
```

Comment:
removing this.data = null will mean that your method ALWAYS goes into the if-part on lines 424+425 - never into the else-part on lines 427 & following. either that else-part can be removed or you now have a logic error.